build/charts/theia/provisioning/datasources/migrate.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function checkDataVersion {
+    tables=$(clickhouse client -h 127.0.0.1 -q "SHOW TABLES")
+    if [[ $tables == *"migrate_version"* ]]; then
+        dataVersion=$(clickhouse client -h 127.0.0.1 -q "SELECT version FROM migrate_version")
+    elif [[ $tables == *"flows"* ]]; then
+        dataVersion="0.1.0"
+    fi
+}
+
+function setDataVersion {
+    tables=$(clickhouse client -h 127.0.0.1 -q "SHOW TABLES")
+    if [[ $tables == *"migrate_version"* ]]; then
+        clickhouse client -h 127.0.0.1 -q "ALTER TABLE migrate_version DELETE WHERE version!=''"
+    else
+        clickhouse client -h 127.0.0.1 -q "CREATE TABLE migrate_version (version String) engine=MergeTree ORDER BY version"
```

Comment:
Does this need to be updated for a clustered database schema since a different engine needs to be used?

Reply:
I think it might be safe to keep as it is, as the table `migrate_version` is not expected to be changed after initialization.  During the initialization, each pod will run this schema management code to update the version independently.